src/client/pages/Parkeren/Parkeren.tsx

```diff
@@ -54,7 +64,14 @@ export default function Parkeren() {
           external={true}
           href="https://parkeervergunningen.amsterdam.nl/landing"
         >
-          Log voor uw vergunning in op Mijn Parkeren
+          Log in op Mijn Parkeren voor uw parkeervergunnning voor bewoners
+        </Linkd>
+        <Linkd
+          external={true}
+          href="https://www.amsterdam.nl/parkeren-verkeer/parkeervergunning/"
+        >
+          Lees hier meer over alle parkeervergunningen in Amsterdam
+          Parkeervergunning

```

Comment:
Dit ziet er raar uit, watskeburt?

Reply:
Dat is prettier die de regel afkapt, maar in de output staat alles netjes op een regel.

![image](https://user-images.githubusercontent.com/92784122/152760460-e825e3db-80fb-4e52-a10b-2e06deb88f1d.png)

Reply:
Ja maar die zin... klopt niet. Wie heeft die bedacht?

Reply:
Copy is helemaal veranderd.